cmd/config/config.go

```diff
@@ -51,11 +44,7 @@ func SetBip44CoinType(config *sdk.Config) {
 
 // RegisterDenoms registers the base and display denominations to the SDK.
 func RegisterDenoms() {
-	if err := sdk.RegisterDenom(DisplayDenom, math.LegacyOneDec()); err != nil {
-		panic(err)
-	}
-
-	if err := sdk.RegisterDenom(BaseDenom, math.LegacyNewDecWithPrec(1, types.BaseDenomUnit)); err != nil {
+	if err := sdk.RegisterDenom(types.DisplayDenom, math.LegacyOneDec()); err != nil {
```

Comment:
Isn't this redundant given the configurator in `app.go`?

Reply:
yeah I don't understand the purpose of this if we have 2 different denominations. 

Either we register both (display and base) or none